.env

```
@@ -5,5 +5,7 @@ CLEAR_ALL_NOTIFICATION_URL="notifications/clear-all"
 MARK_ALL_NOTIFICATION_AS_READ_URL="notifications/mark-all-read"
 DELETE_SINGLE_NOTIFICATION="notifications"
 MARK_NOTIFICATION_AS_READ_URL="notifications/mark-read"
-
+FORGET_PASSWORD_URL="auth/forgotPassword"
```

dotenv-linter (3.3.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)